.travis.yml

```diff
@@ -33,12 +33,20 @@ before_deploy:
 - export PATH="$PATH:$HOME/.rvm/bin"
 - export PHANTOMJS_CDNURL=http://cnpmjs.org/downloads
 deploy:
-  provider: script
-  script: ".travis/deploy.sh"
-  skip_cleanup: true
-  on:
-    repo: WorldBank-Transport/DRIVER
-    branch: develop
+  - provider: script
+    script: ".travis/deploy_staging.sh"
+    skip_cleanup: true
+    on:
+      repo: WorldBank-Transport/DRIVER
+      branch: develop
+  - provider: script
+    script: ".travis/deploy.sh"
+    skip_cleanup: true
+    on:
+      repo: WorldBank-Transport/DRIVER
+      branch: master
+      tags: true
```

Comment:
This may lead to two deployments that race, at least if we use `git-flow`. I think that the completion of a release branch will trigger a build for the merge into `master`, but also a build for the tagged commit.

Reply:
My understanding from the docs is that `branch` and `tags` are AND-ed together such that a build is only kicked off if it satisfies both conditions:
> When _all_ conditions specified in the on: section are met, your build will deploy.

https://docs.travis-ci.com/user/deployment#Conditional-Releases-with-on%3A 

But I have no experience with this aspect of travis so it's definitely possible that that's not quite how it works.

Reply:
Hm. This may be true then. I was basing my understanding off of visually seeing builds get triggered for `master` and tagged commits whenever a release gets finalized (merges into `master`, `develop`, and tagged commit). It is possible that at the end of a build it ANDs these conditions together to make a decision.